mmv1/templates/terraform/examples/compute_rmig_resize_request.tf.tmpl

```diff
@@ -0,0 +1,77 @@
+resource "google_compute_region_instance_template" "a3_dws" {
+  provider = google-beta
+  name                 = "a3-dws"
```

Comment:
This should use a template variable instead of hard-coding the name so that the resource can be cleaned up afterwards (and so that it uses a random name.)

This is currently causing the tests to fail with:

```
Error: Error creating RegionInstanceTemplate: googleapi: Error 409: The resource 'projects/ci-test-project-188019/regions/us-central1/instanceTemplates/a3-dws' already exists, alreadyExists
```

```suggestion
  name                 = "{{index $.Vars "resize_request_name"}}"
```

Reply:
Done!